apps/siiges-app/pages/UserProfile/index.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { Container, Typography } from '@mui/material';
+import { ThemeProvider } from '@mui/material/styles';
+import { MainNavbar, Input, ButtonStyled } from '@siiges-ui/shared';
+import Image from 'next/image';
+import Card from '@mui/material/Card';
+import CardContent from '@mui/material/CardContent';
+import Grid from '@mui/material/Grid';
+import Paper from '@mui/material/Paper';
+import Divider from '@mui/material/Divider';
+import theme from '../theme';
+
+export default function UserProfile() {
+  return (
+    <ThemeProvider theme={theme}>
+      <MainNavbar />
+      <Image
+        alt="background"
+        src="/Fondo.jpg"
+        layout="fill"
+        objectFit="cover"
+        quality={100}
+        style={{
+          zIndex: -1,
+          overflow: 'hidden',
+        }}
+      />
+      <Container>
+        <Card sx={{ minWidth: 275, mt: 5, minHeight: 500 }}>
+          <CardContent>
+            <Grid container spacing={2}>
+              <Grid item xs={4} sx={{ marginTop: 7 }}>
+                <Image
+                  alt="avatar"
+                  src="/avatar.png"
+                  quality={100}
+                  width="300px"
+                  height="300px"
+                  style={{
+                    zIndex: 1,
+                    overflow: 'hidden',
+                  }}
+                />
+                <Paper
+                  sx={{
+                    padding: 2,
+                    marginTop: 3,
+                    width: 300,
+                    textAlign: 'center',
+                  }}
+                >
+                  <Typography variant="p">
+                    Luis Manuel de Alba Villaseñor
+                  </Typography>
+                  <br />
+                  <Divider sx={{ marginY: 1 }} />
+                  <Typography variant="p">
+                    El jefe de jefes
+                  </Typography>
+                </Paper>
```

Comment:
We could create a new component with this
```suggestion
// new-component.js
exports default NewComponent(name, rol) {
        return (
                <Paper
                  sx={{
                    padding: 2,
                    marginTop: 3,
                    width: 300,
                    textAlign: 'center',
                  }}
                >
                  <Typography variant="p">
                    {name}
                  </Typography>
                  <br />
                  <Divider sx={{ marginY: 1 }} />
                  <Typography variant="p">
                    {rol}
                  </Typography>
                </Paper>
        );
}
// UserProfile/index.jsx
<NewComponent name="Luis Manuel de Alba Villaseñor" rol="El jefe de jefes"
```